images/opentelemetry/rootfs/build.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export NGINX_VERSION=1.20.1
```

Comment:
Should we follow this VERSION the same as the nginx been built? Like, we are building 1.19.9 (https://github.com/kubernetes/ingress-nginx/blob/main/images/nginx/rootfs/build.sh#L21), shouldn't we need to use the same?

Is there an easy way we can link those? maybe, an env var? etc?

Anyway I don't think this is a blocking for this first step right now, just thinking in a future when we need to rebuild things, we should make this rebuild more "linked" between all modules :)

Reply:
I used this version from the mod_security branch. But yeah, I agree that we shouldn't have to change the version everywhere just to upgrade it.